packages/airnode-node/src/evm/fulfillments/withdrawals.ts

```diff
@@ -59,13 +59,15 @@ export const submitWithdrawal: SubmitRequest<Withdrawal> = async (airnodeRrp, re
     `Withdrawal gas limit estimated at ${paddedGasLimit.toString()} for Request:${request.id}`
   );
 
-  // We set aside some ETH to pay for the gas of the following transaction,
-  // send all the rest along with the transaction. The contract will direct
-  // these funds back to the sponsor wallet.
+  // We set aside some ETH to pay for the gas of the following transaction and
+  // return the rest to the sponsor wallet less a remainder, if specified.
```

Comment:
Can you also mentioned that this is because some chains require this (e.g. Optimism)

Reply:
Good idea- implemented.